src/scene/materials/standard-material.js

```diff
@@ -40,6 +40,9 @@ Object.assign(pc, function () {
      * * When anisotropy < 0, anistropy direction aligns with the tangent, and specular anisotropy increases as the anisotropy value decreases to minimum of -1.
      * * When anisotropy > 0, anistropy direction aligns with the bi-normal, and specular anisotropy increases as anisotropy value increases to maximum of 1.
      *
+     * @property {number} clearCoat Defines the strength of clear coat layer, clear coat layer is disabled when clearCoat == 0
```

Comment:
What's the range?

Reply:
added range and default - let me know if makes sense now?